src/app/organization/holidays/view-holidays/view-holidays.component.html

```diff
@@ -0,0 +1,61 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary" (click)="activateHoliday()">
+    Activate
+  </button>
+  <button mat-raised-button color="primary" [routerLink]="['edit']">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+  <button mat-raised-button color="warn" (click)="deleteHoliday()">
+    <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+    Delete
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="header">
+          Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.name }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          From Date
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.fromDate | date:'yyyy-MM-dd' }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          To Date
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.toDate | date:'yyyy-MM-dd' }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Repayments Scheduled To
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.repaymentsRescheduledTo | date:'yyyy-MM-dd' }}
+        </div>
+
+      </div>
+
```

Comment:
One more field missing

Reply:
Cqan you mention the field?

Reply:
The field name is the same but the values displayed are different based on the value of the parameter.

Reply:
Found that. But in API doc there's no such field

Reply:
Okay. please follow the community app as well while implementing modules.

Reply:
Done. Please review